test/bats/azure.bats

```diff
@@ -16,17 +16,10 @@ if [ $TEST_WINDOWS ]; then
   NODE_SELECTOR_OS=windows
 fi
 
-if [ -z "$AUTO_ROTATE_SECRET_NAME" ]; then
-    export AUTO_ROTATE_SECRET_NAME=secret-$(openssl rand -hex 6)
-fi
-
-export KEYVAULT_NAME=${KEYVAULT_NAME:-csi-secrets-store-e2e}
+export KEYVAULT_NAME=${KEYVAULT_NAME:-secrets-store-csi-e2e}
 export SECRET_NAME=${KEYVAULT_SECRET_NAME:-secret1}
 export SECRET_VERSION=${KEYVAULT_SECRET_VERSION:-""}
 export SECRET_VALUE=${KEYVAULT_SECRET_VALUE:-"test"}
-export KEY_NAME=${KEYVAULT_KEY_NAME:-key1}
```

Comment:
Trimmed down the azure provider tests because the goal is to have small set of tests for providers. These key specific tests are done in the e2e-provider test suite.